packages/redux/src/exchanges/types/actions.types.ts

```diff
@@ -0,0 +1,111 @@
+import type * as actionTypes from '../actionTypes.js';
+import type { Action } from 'redux';
+import type {
+  BlackoutError,
+  Exchange,
+  ExchangeBookRequest,
+  ExchangeFilters,
+} from '@farfetch/blackout-client';
+
+export interface FetchExchangeRequestAction extends Action {
```

Comment:
These types are not being used anywhere, you should use them in the factory modules to type the dispatch calls.
Please review all action types that are unused in this file and use it in their respective factories.